.github/workflows/tests.yml

```diff
@@ -83,8 +83,8 @@ jobs:
     strategy:
       matrix:
         config:
-          # - {device: "default.qubit", shots: None}
-          # - {device: "default.qubit", shots: 1000}
+          - {device: "default.qubit", shots: None}
+          - {device: "default.qubit", shots: 10000}

```

Comment:
Nice!

Don't forget to also update `codecov.yml`, to increase the `after_n_builds` parameter.

Reply:
Hmm, what does `after_n_builds` do? :thinking: (incremented it)

Reply:
It essentially tells Codecov not to send the coverage report to GitHub until `n` CI builds have been performed. This stops Codecov from sending incorrect coverage updates when not all tests have yet been run.

I see that you've incremented it to 8, but you'll need to increment it to 9, since you are adding two new CI tests here :)

Reply:
Oh nice! Sure, updated now :)